### PR TITLE
[Specification] multifold.instance.json

### PR DESCRIPTION
**Pull request by Cubxity**

### CODE REVIEW

File: specification/0001-multifold.instance.json.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+# multifold.instance.json
+
+## Summary
+
+This document acts as a Request for Comments (RPC) on the topic of how a MultiFold instance metadata is formatted.
+
+## Motivation
+
+This RFC describes a scalable, portable, and more powerful format for the instance that defines what an instance's
```
Comment (KodingDev) [inline specification/0001-multifold.instance.json.md]: Grammar issue. Replace the duplicated instance section.

File: specification/0001-multifold.instance.json.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+The required version of the component. It is recommended to follow [semver](https://semver.org/) specs. It should be
+possible to define a "version range", similar to dependencies in `package.json`. Read more about version
+ranges [here](https://docs.npmjs.com/cli/v6/using-npm/semver).
+</details>
+
+<details> 
+<summary>settings: <code>Record&lt;string, string&gt;?</code></summary>
+
+Record or a map of settings. This is optional and can be freely defined. The options will be passed to the component
```
Comment (KodingDev) [inline specification/0001-multifold.instance.json.md:88]: Example? Some form of additional description would be good.

File: specification/0001-multifold.instance.json.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+upon initialization.
+</details>
+
+## InstanceMod
+
+<details> 
+<summary>id: <code>string</code></summary>
+
+The id of the mod **from the provider**. This is not to be confused with the mod id defined in mcmod.info or
```
Comment (KodingDev) [inline specification/0001-multifold.instance.json.md:97]: Explain further. Would this be for example the CurseForge mod ID in the URL or something else?

**Review by KodingDev — CHANGES_REQUESTED** [review]

File: specification/0001-multifold.instance.json.md (hunk 2 above)
Comment (Cubxity) [reply specification/0001-multifold.instance.json.md:88]: Example of this is already provided in the example section.

**Review by KodingDev — APPROVED** [review]

**Review by proudmuslim-dev — COMMENTED** [review]
formats, such as* (https://github.com/MultifoldLauncher/rfcs/pull/1/files#diff-83d178283bdcfa1508b3abebd57ebcd51ee5173c8c6a47e9a0bda9ea90568ebdR15)

**Review by proudmuslim-dev — COMMENTED** [review]
There _are_ no good alternatives* (https://github.com/MultifoldLauncher/rfcs/pull/1/files#diff-83d178283bdcfa1508b3abebd57ebcd51ee5173c8c6a47e9a0bda9ea90568ebdR164)

File: specification/0001-multifold.instance.json.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+This document acts as a Request for Comments (RPC) on the topic of how a MultiFold instance metadata is formatted.
+
+## Motivation
+
+This RFC describes a scalable, portable, and more powerful format that defines an instance. MultiFold offers users the
+ability install mod loaders and mods seamlessly, without manual configuration.
+
+## Explanation
+
+The following is the schema for instance metadata. The primary format is JSON, however it can be represented in other
```
Comment (KodingDev) [inline specification/0001-multifold.instance.json.md:14]: I still believe there is no need for any formats aside from JSON, only further complicating the internals. Not an issue that it's here, but in my opinion should be reconsidered.

**Review by KodingDev — APPROVED** [review]

File: specification/0001-multifold.instance.json.md (hunk 4 above)
Comment (Cubxity) [reply specification/0001-multifold.instance.json.md:14]: The point here is that the schema can be used/represented in other formats, the launcher itself is going to read and write in JSON.

Comment (KodingDev) [reply specification/0001-multifold.instance.json.md:14]: Perfectly reasonable, all good.

**Pull request merged**